src/components/fields/image-upload/image-input/file-item/file-item.tsx

```diff
@@ -170,7 +171,7 @@ export const FileItem = ({ id = "file-item", index, fileItem, maxSizeInKb, accep
 						data-testid={TestHelper.generateId(`${id}-${index + 1}`, "file-image")}
 						ref={fileNameWrapperRef}
 					>
-						{transformedFileName}
+						<FilenameWrapper>{transformedFileName}</FilenameWrapper>
```

Comment:
- assign `fileNameWrapperRef` to `FilenameWrapper`
- rename either the element or the ref to use the same casing

Reply:
updated